### PR TITLE
OCPBUGS-27323: Skip tests for image-registry operator with single replica

### PR DESCRIPTION
**Pull request by Shilpa-Gokul**
Fix for https://issues.redhat.com/browse/OCPBUGS-27323

### CODE REVIEW

**Comment by bparees** [comment]
/unassign

/assign @stbenjam @dgoodwin 

@stbenjam @dgoodwin @deads2k owners file is badlly out of date, would you like to remove it entirely and defer to the top level owners file, or update it?

**Review by djslavens — APPROVED** [review]
/lgtm

File: pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go [hunk 1]
```diff
@@ -136,7 +156,10 @@ func testOperatorStateTransitions(events monitorapi.Intervals, conditionTypes []
 
 			excepted := []string{}
 			fatal := []string{}
-
+			var replicaCount int32
```
[the remark this answers is not in the document]
Comment (Shilpa-Gokul) [reply pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go]: @xueqzhan Addressed your comments

File: pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go [hunk 2]
```diff
@@ -38,8 +40,8 @@ func testStableSystemOperatorStateTransitions(events monitorapi.Intervals) []*ju
 	return testOperatorStateTransitions(events, []configv1.ClusterStatusConditionType{configv1.OperatorAvailable, configv1.OperatorDegraded}, except)
 }
 
-func testUpgradeOperatorStateTransitions(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
-	except := func(operator string, condition *configv1.ClusterOperatorStatusCondition) (string, error) {
+func testUpgradeOperatorStateTransitions(events monitorapi.Intervals, clientConfig *rest.Config) []*junitapi.JUnitTestCase {
+	except := func(operator string, condition *configv1.ClusterOperatorStatusCondition, clientConfig ...*rest.Config) (string, error) {
```
Comment (xueqzhan) [inline pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go:44]: I might be missing something here. Why do we need multiple clientConfig here?

Comment (Shilpa-Gokul) [reply pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go]: I had defined that as a variadic function to minimise the code changes required in testOperatorStateTransitions code, have changed it to another approach now, please check

**Pull request merged**